src/src/components/collections/collections.jsx

```diff
@@ -84,9 +84,9 @@ export function CollectionForm (props){
       var UUIDs = [];
       if (editingCollection.datasets && editingCollection.datasets.length > 0) {
         for (const entity of editingCollection.datasets) {
-          //When coming from the Entity, the Datasets use data_types, from the Search UI they pass display_subtype instead
-          if (entity.data_types && entity.data_types.length > 0) {
-            var subtype = generateDisplaySubtypeSimple_UBKG(entity.data_types[0],props.dtl_all);
+          //When coming from the Entity, the Datasets use dataset_type, from the Search UI they pass display_subtype instead
+          if (entity.dataset_type && entity.dataset_type.length > 0) {
```

Comment:
The `dataset_type` is a string field, unlike `data_types` which is an array, so should not check the size and use `dataset_type[0]`

Reply:
the length check Helps make sure its not empty